connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Worker.java

```diff
@@ -983,7 +983,7 @@ private List<ErrorReporter> sinkTaskReporters(ConnectorTaskId id, SinkConnectorC
         if (topic != null && !topic.isEmpty()) {
             Map<String, Object> producerProps = baseProducerConfigs(id.connector(), "connector-dlq-producer-" + id, config, connConfig, connectorClass,
                                                                 connectorClientConfigOverridePolicy, kafkaClusterId);
-            Map<String, Object> adminProps = adminConfigs(id.connector(), "connector-dlq-adminclient-", config, connConfig, connectorClass, connectorClientConfigOverridePolicy, kafkaClusterId, ConnectorType.SINK);
+            Map<String, Object> adminProps = adminConfigs(id.connector(), "connector-dlq-adminclient-" + id, config, connConfig, connectorClass, connectorClientConfigOverridePolicy, kafkaClusterId, ConnectorType.SINK);
```

Comment:
Small unrelated fix spotted while I was in the neighborhood. I checked the rest of this file for similar bugs; this appears to be the only one.